rpc/eth/jsonrpc_conversion.go

```diff
@@ -188,6 +187,13 @@ func EncEvent(log types.EventData) JsonLog {
 	return jLog
 }
 
+func EncSubscriptionEvent(log types.EventData) JsonLog {
```

Comment:
Since this 3-liner appears to be only called from one place just inline the code at the callsite.